com.github.eneshecan.WhatsApp-For-Linux.yml

```
@@ -0,0 +1,146 @@
+app-id: com.github.eneshecan.WhatsApp-For-Linux
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: whatsapp-for-linux
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+# - --device=dri # OpenGL rendering
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=com.github.eneshecan.whatsapp-for-linux
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos 
+rename-desktop-file: whatsapp-for-linux.desktop
+rename-icon: whatsapp-for-linux
+cleanup:
+  - "/include"
+  - "*.la"
+modules:
+  - name: mm-common
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/mm-common/1.0/mm-common-1.0.4.tar.xz
+        sha256: e954c09b4309a7ef93e13b69260acdc5738c907477eb381b78bb1e414ee6dbd8
+    cleanup:
+      - '*'
+
+  - name: sigc++-2
+    config-opts:
+      - --disable-documentation
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsigc++/2.10/libsigc++-2.10.8.tar.xz
+        sha256: 235a40bec7346c7b82b6a8caae0456353dc06e71f14bc414bcc858af1838719a
+
+  - name: cairomm
+    config-opts:
+      - --disable-documentation
+    cleanup:
+      - /lib/cairomm-1.0
+    sources:
+      - type: archive
+        url: https://www.cairographics.org/releases/cairomm-1.14.3.tar.xz
+        sha256: 0d37e067c5c4ca7808b7ceddabfe1932c5bd2a750ad64fb321e1213536297e78
+
+  - name: glibmm
+    cleanup:
+      - /lib/glibmm-2.4
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+      - -Dbuild-examples=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.2.tar.xz
+        sha256: b2a4cd7b9ae987794cbb5a1becc10cecb65182b9bb841868625d6bbb123edb1d
+
+  - name: pangomm
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.1.tar.xz
+        sha256: c885013fe61a4c5117fda395770d507563411c63e49f4a3aced4c9efe34d9975
```

```suggestion
        url: https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.2.tar.xz
        sha256: 57442ab4dc043877bfe3839915731ab2d693fc6634a71614422fb530c9eaa6f4
```

You need 2.46.2 to build.